index.html

```diff
@@ -38,6 +39,131 @@ <h1 class="primary-text">I’m Richmond Glad to see you!</h1>
+            <p class="card-title">Tonic</p>
+            <div class="tag">
+              <ul>
+                <li>Canopy</li>
+                <li>Backend-Dev</li>
+                <li> 2015</li>
+            </ul>
+             </div>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+          <div class="card">
+            <div class="img"><img src="./logos/t4.png" alt="work example" /></div>
+            <p class="card-title">Multi-Post Stories</p>
+            <div class="tag">
+              <ul>
+                <li>Canopy</li>
+                <li>Backend-Dev</li>
+                <li> 2015</li>
+            </ul>
+             </div>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+        </div>
+      </section>
+      <section class="container2">
+        <p class="primary-text">About Me</p>
+        <p class="content">
+            I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <p class="Connect">LET'S CONNECT</p>
+        <ul class="social-media">
+            <li class="medium"><a href="#"><img src="./logos/medium.png" alt="medium" /></a>
+            </li>
+            <li class="github"><a href="#"><img src="./logos/github.png" alt="github" /></a>
+            </li>
+            <li class="angelistic"><a href="#"><img src="./logos/angelist.png" alt="angelistic" /></a>
+            </li>
+            </ul>
+             <div class="button1"><a href="#">Get My Resume</a></div>
+
+            <div class="skill-title">
+              <div class="container1">
+                <h1 class="skill">Languages</h1>
+                <h1 class="dropdown">--</h1>
+              </div>
+             <div class="container1">
+              <div class="javascript"><img src="./logos/javascript2.png" alt="github-logo"/>Javascript</div>
+              <div class="html"><img src="./logos/html.png" alt="github-logo"/>Html</div>
+              <div class="css"><img src="./logos/css.png" alt="github-logo"/>Css</div>
+              <h2 class="skill2">Frameworks</h2>
+
+              <h2 class="skill2">Skills</h2>
+             </div> 
+              
+          </div>
```

Comment:
- [ ] Please use `ul` and `li` for the technology categories ("Languages", "Frameworks" and "Skills") and each category(e.g., "JavasScript", "HTML" and "CSS") also should be in a list `ul` and each of them a `li`.